src/java/org/apache/cassandra/dht/BootStrapper.java

```diff
@@ -59,6 +69,14 @@ public class BootStrapper extends ProgressEventNotifierSupport
     private final MovementMap movements;
     private final MovementMap strictMovements;
 
+    static
```

Comment:
is this ... ok? why cant we put this to e.g. init()

Reply:
The metrics should only be registered once, when the class is inited. However, since it relies on the local counters, e.g. `bootstrapFilesTotal`, it is easier to register them as static variables.